### PR TITLE
Improve acceptance tests run against remote instances

### LINKED ISSUE #243: Acceptance tests remote run have missing context logic
### Problem

Currently the acceptance tests are missing context such as
- unique log heaader
- account balance consumed
- config details

### Solution

Updated acceptance tests with details to improve remote runs

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by Nana-EC**
Signed-off-by: Nana-EC <nana@swirldslabs.com>

**Description**:
Improve acceptance test runs
- Update `rpc.spect.ts`
  - differentiate logs
  - increase account1 balance to afford other transactions
  - log configurations
  - display test expense
- Shorten client logs
- Fix missed metric removal logic
- Trim `OPERATOR_ID_MAIN`
- Add timeout reset on mirror client
- add operator balance check

**Related issue(s)**:

Fixes #243 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/server/tests/acceptance/rpc.spec.ts [hunk 1]
```diff
@@ -234,8 +247,8 @@ describe('RPC Server Acceptance Tests', function() {
 
         const defaultLegacyTransactionData = {
             value: ONE_TINYBAR,
-            gasPrice: 720000000000,
-            gasLimit: 3000000
+            gasPrice: 7200000000000,
+            gasLimit: 30000000
```
Comment (shemnon) [inline packages/server/tests/acceptance/rpc.spec.ts:251]:
```suggestion
            gasPrice: 7_200_000_000_000,
            gasLimit: 30_000_000
```

File: packages/server/tests/acceptance/rpc.spec.ts [hunk 2]
```diff
@@ -246,54 +259,54 @@ describe('RPC Server Acceptance Tests', function() {
         const defaultLondonTransactionData = {
             value: ONE_TINYBAR,
             chainId: Number(CHAIN_ID),
-            maxPriorityFeePerGas: 720000000000,
-            maxFeePerGas: 720000000000,
+            maxPriorityFeePerGas: 7200000000000,
+            maxFeePerGas: 7200000000000,
```
Comment (shemnon) [inline packages/server/tests/acceptance/rpc.spec.ts:263]:
```suggestion
            maxPriorityFeePerGas: 7_200_000_000_000,
            maxFeePerGas: 7_200_000_000_000
```

File: packages/server/tests/acceptance/rpc.spec.ts [hunk 3]
```diff
@@ -246,54 +259,54 @@ describe('RPC Server Acceptance Tests', function() {
...
             type: 2
         };
 
         const defaultLegacy2930TransactionData = {
             value: ONE_TINYBAR,
             chainId: Number(CHAIN_ID),
-            gasPrice: 720000000000,
-            gasLimit: 3000000,
+            gasPrice: 7200000000000,
+            gasLimit: 30000000,
```
Comment (shemnon) [inline packages/server/tests/acceptance/rpc.spec.ts:272]:
```suggestion
            gasPrice: 7_200_000_000_000,
            gasLimit: 30_000_000
```

**Review by shemnon — APPROVED** [review]

File: packages/server/tests/acceptance/rpc.spec.ts (hunk 1 above)
Comment (Nana-EC) [reply packages/server/tests/acceptance/rpc.spec.ts]: Updated

File: packages/server/tests/acceptance/rpc.spec.ts (hunk 2 above)
Comment (Nana-EC) [reply packages/server/tests/acceptance/rpc.spec.ts]: Updated

File: packages/server/tests/acceptance/rpc.spec.ts (hunk 3 above)
Comment (Nana-EC) [reply packages/server/tests/acceptance/rpc.spec.ts]: Updated

**Review by shemnon — APPROVED** [review]

**Pull request merged**